documentation/TIPS.md

```diff
@@ -0,0 +1,38 @@
+# Welcome to OpenCircuits!
+
+I wanted to create this document to help you have a seamless transition into the team and answer some questions that you may have. Firstly, you may be wondering what OpenCircuits is. 
+OpenCircuits is an open-source circuit designer that was created by Leon Montealegre, who is the program manager. The team is also currently in the process of creating OpenCircuitsEE which is an open-source circuit designer geared towards electrical engineering and computer systems engineering students at RPI. We are hoping that OpenCircuitsEE will replace the current LTspice programs being used in the ECSE curriculum. You will have the freedom to work on either project throughout the semester.
+
+You should start off by navigating through the OpenCircuits website and familiarizing yourself with the tools (bonus points if you can find any bugs (http://opencircuits.io/).
+
+In OpenCircuits, you will be using GitHub which is a repository hosting service. If you are unfamiliar with GitHub, don’t be shy to ask Leon or other team members about how to navigate through Github! (https://github.com/OpenCircuits/OpenCircuits). 
+
+In order to get set up, make sure to access the OpenCircuits wiki page and follow the instructions to get set up (https://github.com/OpenCircuits/OpenCircuits/wiki).
+
+I would suggest downloading Visual Studio 2 to use are your Integrated Development Environment (IDE).
+
+Don’t be afraid to ask around! We are all learning together and we’re here to help!
+
+Happy Coding!
+-OpenCircuiits Team
+
+# Tips:
+If you are unfamiliar with Visual Studio 2, I would highly suggest watching this video to get familiar with the IDE: https://www.youtube.com/watch?v=BpSbKmZkKFQ
+
+* If you are unfamiliar with CSS or HTML (the backbone languages of OpenCircuits), I would suggest going through CodeAcademy to get familiar: https://www.codecademy.com/
```

Comment:
> CSS or HTML (the backbone languages of OpenCircuits)

Opencircuits is 88.6% TypeScript and the backend is in GoLang, so I would recommend resources for those, too